cpp/models/abm/location.cpp

```diff
@@ -99,12 +98,12 @@ void Location::begin_step(TimeSpan /*dt*/, const GlobalInfectionParameters& glob
         m_cached_exposure_rate = {{AgeGroup::Count, VaccinationState::Count}, 0.};
     }
     else if (m_cells.empty()) {
-        auto num_carriers              = get_subpopulation(InfectionState::Carrier);
-        auto num_infected              = get_subpopulation(InfectionState::Infected);
+        auto num_infected_no_symptoms              = get_subpopulation(InfectionState::InfectedNoSymptoms);
+        auto num_infected_symptoms              = get_subpopulation(InfectionState::InfectedSymptoms);
```

Comment:
These helper names (as they differ from the original one) actually lead to problems if we want to replace names in all files.

We should maybe avoid them if they do not really improve readibility or reduce the number of code lines / line length. Here, we could direct use `get_subpopulation` below, couldnt we?

Reply:
If you have suggestions for these names (and also the other names such as `SusceptibleToExposedByInfectedNoSymptoms`) please feel free to share them because I also don't really like them.

Reply:
I sometimes went for using the same name just without "InfectionState::" and starting with a small letter, i.e. `infectedNoSymptoms`. This does not help for refactoring but at least you find all appearances at once then.

Here you can maybe use the get_subpopulation() directly where you need the numbers?

Reply:
Used get_subpopulation() directly.